src/main/antora/modules/websvcs/pages/rest/rest003.adoc

```diff
@@ -8,13 +8,13 @@ This section demonstrates the steps that are needed to create, build, deploy, an
 This section explains how to use NetBeans IDE to create a RESTful web service using a Maven archetype.
 The archetype generates a skeleton for the application, and you simply need to implement the appropriate method.
 
-You can find a version of this application at `_jakartaee-examples_/tutorial/jaxrs/hello/`.
+You can find a version of this application at `_jakartaee-examples_/tutorial/rest/hello/`.
```

Comment:
The references in here point to the jakartaee-examples repo: https://github.com/eclipse-ee4j/jakartaee-examples. Can you raise a PR for that repo as well to change the folder names?

Reply:
Okay, I will raise PR for that.